src/Bot/commands/index.ts

```diff
@@ -1,6 +1,19 @@
-import { Awaitable, Interaction } from "discord.js";
+import {
+  Awaitable,
+  ChatInputCommandInteraction,
+  Interaction,
+} from "discord.js";
 import { assistantAskReply, lookupReply } from "./commandsReplies";
 
+const runIfChatInputCommand = (
+  interaction: Interaction,
+  callback: (interaction: ChatInputCommandInteraction) => Promise<void>
+) => {
+  if (interaction.isChatInputCommand()) {
```

Comment:
I added to narrow the types to `ChatInputCommandInteraction` because this is more accurate than just `Interaction`.